pkg/apis/admissionregistration/types.go

```diff
@@ -334,6 +339,18 @@ type Validation struct {
 	MessageExpression string
 }
 
+// Variable is the definition of a variable that is used for composition.
+// +structType=atomic
+type Variable struct {
+	// Name is the name of the variable. The variable can be accessed in other expressions through `variables`
+	// For example, if name is "foo", the variable will be available as `variables.foo`
```

Comment:
Clarify that the variable name must be a valid CEL identifier?

Reply:
added.